.github/workflows/build-plugin-archive.yml

```
@@ -35,7 +35,7 @@ on:
         type: string
       PHP_VERSION_BUILD:
         description: PHP version to use when executing build tools.
-        default: "8.0"
+        default: "8.2"
```

Can we leave the PHP update to the PR that touches all files, #108?
```suggestion
        default: "8.0"
```